ocaml/xapi/xapi_xenops.ml

```diff
@@ -741,44 +742,24 @@ module MD = struct
 
   let vgpus_of_vm ~__context (vmref, vm) =
     let open Vgpu in
-    if Vgpuops.vgpu_manual_setup_of_vm vm
```

Comment:
The function `Vgpuops.vgpu_manual_setup_of_vm` is now unused, so please delete the definition.

Reply:
OK, will remove.